### ISSUE TITLE
Can you push a new version to Pypi?

### ISSUE THREAD

**Issue by rich-bain** [jcrobak/parquet-python#73]
Hi,
Thank you for the great project. 

The current version on Pypi doesn't have 2 already implemented fixes that I need for a pure-python stack (#69 / #70 and #64).

Are you able to push a new version to Pypi?

**Comment by edisongustavo** [comment]
Yes please, I'm interested in a new release too!

**Comment by Saluev** [comment]
Moveover, current wheel in pypi still depends on old `thriftpy` package:
```
Requires-Dist: python-snappy
Requires-Dist: thriftpy (>=0.3.6)
```
@jcrobak can you upload a new build to pypi please?

**Comment by jcrobak** [comment]
I've published version 1.3.1 - hopefully that works for you! https://pypi.org/project/parquet/
